src/test/java/AndreeaG/tests/tema6/HerokuAlerts.java

```diff
@@ -0,0 +1,101 @@
+package AndreeaG.tests.tema6;
+
+import AndreeaG.driver.BrowserManager;
+import AndreeaG.utils.FileUtils;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchWindowException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.io.IOException;
+import java.util.NoSuchElementException;
+
+public class HerokuAlerts {
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws IOException {
+        navigateToHerokuHomePage();
+        try {
+            openAlertsPage();
+            openFirstAlertOnPage();
+            acceptAlertOnPage();
+            openAlert();
+            acceptAlert();
+            thirdAlert();
+            acceptThirdAlert();
+            closeBrowser();
+
+        } catch (NoSuchElementException e) {
+            FileUtils.takeScreenshot(driver, "alert");
+        } catch (NoSuchWindowException e) {
```

Comment:
Replace NoSuchWindowException with NoSuchAlertException as we are only opening alerts. Thanks!